specification/containerservice/resource-manager/readme.go.md

```diff
@@ -20,6 +20,7 @@ azure-arm: true
 
 ``` yaml $(go) && $(multiapi)
 batch:
+  - tag: package-2021-07
```

Comment:
is it possible to also change the default tag to this in `readme.md`? Therefore the track 2 SDKs will get updated to the new version

Reply:
Hi, I think the default tag has been changed in this [commit](https://github.com/Azure/azure-rest-api-specs/commit/54ec19a2c14f085f87e910b36999b11afa49123b), which is auto generated when creating this dev branch.